lib/connect-session-sequelize.js

```diff
@@ -141,7 +142,7 @@ module.exports = function SequelizeSessionInit (Store) {
 
   SequelizeStore.prototype.clearExpiredSessions = function clearExpiredSessions (fn) {
     debug('CLEARING EXPIRED SESSIONS')
-    return this.sessionModel.destroy({where: {'expires': {lt: new Date()}}}).asCallback(fn)
+    return this.sessionModel.destroy({where: {'expires': {[Op.lt || 'lt']: new Date()}}}).asCallback(fn)
```

Comment:
this is possible only in ES6 and I accept all node versions at the moment. I guess we can safely increase the node requirement soon but I'll have to double check this. Do you know which node version sequelize requires?

Reply:
Ah. Offcourse... I have been sitting to long with babel and es6.
To answer your question sequelize requires "node": ">=4.0.0".

Do you want me to make a new pull request that is supported by node 4.0.0?

Reply:
After looking more into node versions. Which part of the code would not be able to run on node v4? I have tried the square brackets syntax on nodejs v4.2.6 which works.

Reply:
oh, really? I thought it doesn't :D ok well, I'll have a look tonight and then merge + publish. Sorry for that.

Reply:
No probs. I also haven't used that syntax before. Pretty powerfull actually. :) Thanks!

Reply:
ah, yeah it doesn't work on 0.10 but that's probably ok to get rid of by now ;)